policy/modules/services/console_setup.if

```diff
@@ -0,0 +1,48 @@
+## <summary>console font and keymap setup program for debian</summary>
+
+########################################
+## <summary>
+##  Execute console-setup in the console_setup domain.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain allowed to transition.
+##	</summary>
+## </param>
+#
+interface(`console_setup_domtrans', `
+    gen_require(`
+        type console_setup_t, console_setup_exec_t;
+        type console_setup_conf_t;
+        type console_setup_runtime_t;
+    ')
+
+    corecmd_search_bin($1)
+    domtrans_pattern($1, console_setup_exec_t, console_setup_t)
+
+    allow $1 console_setup_conf_t:dir search;
+    allow $1 console_setup_conf_t:file read_file_perms;
+    allow $1 console_setup_runtime_t:dir search;
+    allow $1 console_setup_runtime_t:file getattr;
```

Comment:
This should not be here. Not every potential user of the interface will need this.